model/channel_member.go

```diff
@@ -146,11 +153,16 @@ func IsSendEmailValid(sendEmail string) bool {
 	return sendEmail == CHANNEL_NOTIFY_DEFAULT || sendEmail == "true" || sendEmail == "false"
 }
 
+func IsIgnoreChannelMentionsValid(ignoreChannelMentions string) bool {
+	return ignoreChannelMentions == "ignore_channel_mentions_on" || ignoreChannelMentions == "ignore_channel_mentions_off" || ignoreChannelMentions == "ignore_channel_mentions_not_set"
```

Comment:
Could you add constants for all 3 possible values and use them everywhere?

Reply:
👌🏽 will change that